src/main/java/com/afornalik/ox/model/player/PlayerContainer.java

```diff
@@ -0,0 +1,35 @@
+package com.afornalik.ox.model.player;
+
+import com.afornalik.ox.model.board.FieldStatus;
+
+import java.util.Map;
+
+public class PlayerContainer {
+
+    private Player playerOne;
+    private Player playerTwo;
+
+    public void createPlayer(Map<String, Object> playerInfo) {
+        if(playerOne == null) {
+            playerOne = convertMapToPlayer(playerInfo);
+        }else if(playerTwo == null) {
+           playerTwo = convertMapToPlayer(playerInfo);
+        }
+    }
+
+    private Player convertMapToPlayer(Map<String, Object> playerInfo) {
+        return new Player.PlayerBuilder((FieldStatus)playerInfo.get("FieldStatus"))
+                .name((String)playerInfo.get("name"))
+                .score((int)playerInfo.get("score"))
+                .first((boolean)playerInfo.get("first"))
+                .build();
+    }
+
+    public String showPlayerOne() {
```

Comment:
What if one day you'd have to make three players? Making this method generic (only taking player id or something) would make code not only more resyliant to change but would also remove code dupplications in other parts of app where you have to explicitly call one or other method.

Reply:
Changed player type to generic list.